### PR TITLE
Editor can select location

### PR DESCRIPTION
**Pull request by PaulineBA**
### User Story
```
As an editor
In order to give local news to the client
I would like to be able to associate the article with the location

```
## Screenshots (Client)
Screen Shot 2020-06-03 at 2 26 18 PM

### CODE REVIEW

File: src/components/PublishArticle.jsx [hunk 1]
```diff
@@ -23,6 +23,8 @@ const PublishArticle = (props) => {
           category: document
             .getElementById("category")
             .firstElementChild.innerText.toLowerCase(),
+          location: e.target.local.checked && "Sweden",
```
Comment (erikbjoern) [inline src/components/PublishArticle.jsx:26]: Looks clever. Have you checked with a debugger what param is sent off?

**Review by designerofthing — APPROVED** [review]

**Review by erikbjoern — CHANGES_REQUESTED** [review]
We need the form to be cleared when the editor has published the article

File: src/components/PublishArticle.jsx (hunk 1 above)
Comment (erikbjoern) [reply src/components/PublishArticle.jsx:26]: I checked now, in the local API log, we receive the param `location: "Sweden` 😎  Nice job!

**Comment by erikbjoern** [comment]
~~Can you also add a fix for the Review layout? The image appears behind the 'published at' and the body, I guess that's because we increased the image size.~~

^ This can be done in a separate feature though.

**Review by kermit-klein — APPROVED** [review]

**Review by faraznaeem — APPROVED** [review]

**Review by ViaMarcus — APPROVED** [review]

**Comment by kermit-klein** [comment]
Can we merge this ?

**Comment by erikbjoern** [comment]
Replaced by PR#16

**Pull request closed without merging**